docs/usage.md

```diff
@@ -9,36 +9,66 @@ The FortiSOAR Global Visibility Solution Pack provides a comprehensive set of to
+
+Once scenario is executed, mock records will created and it will look like below;
+    
+![Global Visibility Scenario](./res/global-visibility-data-01.png)
+ 
+
 ## Dashboard
-This section continues to describe the remaining widgets in the FortiSOAR Global Visibility Solution Pack, along with their functionalities, interactions, and best practices.
+This section continues to describe the dashboard and remaining widgets in the FortiSOAR Global Visibility Solution Pack, along with their functionalities, interactions, and best practices.
 
-### Record Summary Card
-As described earlier, the Record Summary Card widget provides an overview of record metrics for each site. Additionally, it has the unique ability to broadcast an event when clicked. This event triggers data updates in other widgets.
+![Global Visibility Scenario](./res/global-visibility-dashboard-00.png)
 
-#### Broadcast Event
+### Global Visibility Dashboard Edit View
+This section describes the edit view of FortiSOAR Global Visibility Dashboard. You can visualize different widgets added and configured in this dashboard.
 
-When a user clicks on a Record Summary Card representing a site, the widget broadcasts an event containing information about the selected site. This event serves as a trigger for other widgets to adjust their displayed data according to the selected site.
+![Global Visibility Scenario](./res/global-visibility-dashboard-edit-01.png)
 
 ### Record Summary Tile
 The Record Summary Tile widget offers a more detailed breakdown of record metrics for all sites combined. It complements the Record Summary Card by providing a compact visualization of metrics such as Total Alerts, Total Indicators, Total Tenants.
 
+![Global Visibility Scenario](./res/global-visibility-dashboard-widget-01.png)
+
+### Record Summary Card
+As described earlier, the Record Summary Card widget provides an overview of record metrics for each site. Additionally, it has the unique ability to broadcast an event when clicked. This event triggers data updates in other widgets.
+
+![Global Visibility Scenario](./res/global-visibility-dashboard-widget-02.png)
+
+#### Broadcast Event
+When a user clicks on a Record Summary Card representing a site, the widget broadcasts an event containing information about the selected site. This event serves as a trigger for other widgets to adjust their displayed data according to the selected site.
 
 ### Funnel Chart
 The Funnel Chart widget visualizes the progression of records through various stages of the workflow. It's an excellent tool for understanding the efficiency of your incident response process.
 
+![Global Visibility Scenario](./res/global-visibility-dashboard-widget-04.png)
+
 #### Event Listening
 
 The Funnel Chart widget listens for the broadcasted event triggered by the Record Summary Card. When the event is received, the Funnel Chart updates to show the funnel for the selected site.
```

Comment:
The **Funnel Chart** widget listens for the broadcasted event triggered by the _Record Summary Card_. When the event is received, the **Funnel Chart** updates to show the data in a funnel visualization for the selected site.